OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -3538,91 +3545,244 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="right">35</dim>
                                         <dim type="intercolumn">14</dim>
                                     </dimensions>
-                                    <ab type="ruling" subtype="pattern">Ruling pattern: 1A-1A-1A1A/0-0/0-0/C.</ab>
+                                    <note>The characters per column are 8–9.</note>
+                                    <ab type="pricking">Ruling is visible; pricking is visible.</ab>
+                                    <ab type="ruling" subtype="pattern">
+                                        <!--several, but not all folia containing exclusively miniatures are pricked, but not ruled-->
+                                        <locus from="1" to="2v"/>
+                                        <locus from="7" to="19"/>
+                                        <locus from="21" to="121"/>Ruling pattern: 1A-1A-1A1A/0-0/0-0/C.</ab>
+                                    <ab type="pricking">Primary pricks are visible.</ab>
+                                    <ab type="pricking">Ruling pricks are visible.</ab>
+                                    <ab type="ruling">The upper line is written above the
+                                        ruling.</ab>
+                                    <ab type="ruling">The bottom line is written above the
+                                        ruling.</ab>
                                 </layout>
                             </layoutDesc>
                             
                         </objectDesc>
                         
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
+                                <date notBefore="1600" notAfter="1799"/>
+                                <desc>Fine and regular handwriting. Regularly spaced characters.</desc>
                            <seg type="ink">Black, red.</seg>
                                 <seg type="rubrication">
                                     Incipits (one word to one or several lines, consecutive or alternating with black lines in different groupings), 
                                     elements of punctuation signs, numerals and text dividers,
                                     holy names (<persName ref="PRS6819Mary"/>, <persName ref="PRS4377Gabriel"/>, 
-                                    rarely <persName ref="PRS5684JesusCh"/>), the names of <persName ref="PRS12911KidanaK"/>,
+                                    rarely <persName ref="PRS5684JesusCh"/>), the names of individuals related to the production of 
+                                    the manuscript (<persName ref="PRS12911KidanaK"/>,
                                     <persName ref="PRS13031WaldaA"/>, <persName ref="PRS12964Gorgoryos"/>. 
```

Comment:
do you need the dot at the end here?

Reply:
It's a style question, I would say. I am just used to it.

Reply:
Sorry, the dot after Gorgoryos was a mistake I missed. I was talking about the one after miniatures.